OPHD/Map/Tile.h

```diff
@@ -25,11 +25,11 @@ class Tile
 
 public:
 	Tile() = default;
-	Tile(NAS2D::Point<int> position, int depth, TerrainType index);
-	Tile(const Tile& other) = delete;
-	Tile& operator=(const Tile& other) = delete;
-	Tile(Tile&& other) noexcept;
-	Tile& operator=(Tile&& other) noexcept;
+	Tile(NAS2D::Point<int>, int, TerrainType);
+	Tile(const Tile&) = delete;
+	Tile& operator=(const Tile&) = delete;
+	Tile(Tile&&) noexcept;
+	Tile& operator=(Tile&&) noexcept;
```

Comment:
I find this makes the code less readable. I know parameter names don't need to be included in the header file, though they can be really useful as a form of documentation. I would encourage always including parameter names in header files.

Reply:
Alright. I was starting to opt to keep them out of headers but I forget not everybody is using Visual Studio or VS Code. I'll move in the direction of leaving them in the headers.